community/vmwh/sources

```diff
@@ -0,0 +1,5 @@
+git+https://github.com/jcs/vmwh#ada51feced4b4f203ec725264b893dee9c8930f7
```

Comment:
Drop the `#commit` portion. This software is likely to never get another commit.

Reply:
Done.